src/wrapper/data_chunk.ml

```diff
@@ -29,6 +29,28 @@ let get_opt t type_ idx =
 
 let free t ~here = Resource.free t.data_chunk ~here
 
+let to_string_hum ?(bars = `Unicode) ~column_count t =
+  if t.length = 0 || column_count = 0
+  then ""
+  else (
+    let columns =
+      List.init column_count ~f:(fun idx ->
+        let name = sprintf "Column %d" idx in
+        Ascii_table_kernel.Column.create name (fun i ->
+          if i < t.length
+          then (
+            let _ =
```

Comment:
Fetching a vector, then ignoring it does not constitute getting values. Let's make sure that we actually extract values and convert them to human-readable strings. Refer to Result_.to_string_hum for inspiration.